File: Backend/App/models/menstural_history.py
from sqlalchemy import Column,String,Integer,Boolean,ForeignKey
from Backend.App.core.database import Base
from Backend.App.models.case_history import CaseHistory

class MensturalHistory(Base):
    __tablename__ ="Menstural_History"

    MensturalHistoryId = Column(Integer,primary_key=True,index=True)
    CaseId=Column(Integer,ForeignKey(CaseHistory.CaseId))
    LMP=Column(String(255))
    Duration = Column(String(255))
    Flow = Column(String(255))
    Clots = Column(String(255))
    Cycle = Column(String(255))
    Others = Column(String(255))

File: DotNetBackend/ViveksHomoeoClinic/ViveksHomoeoClinic/Models/EFModel/MensturalHistory.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ViveksHomoeoClinic.Models.EFModel
{
    public class MensturalHistory
    {
        [Key]
        public int MensturalHistoryId { get; set; }
        [Required]
        public int CaseId { get; set; }

        [ForeignKey(nameof(CaseId))]
        public CaseHistory Case { get; set; }
        public string LMP { get; set; }
        public string Duration { get; set; }
        public string Flow { get; set; }
        public string Clots { get; set; }
        public string Cycle { get; set; }
        public string Others { get; set; }
    }
}
